### ISSUE TITLE
Create a disclaimer to avoid liability for Buoy, web admins & users

### ISSUE THREAD

**Issue by christyleos** [betterangels/buoy#164]
This issue was brought up by institutional support providers, specifically shelter, legal aid, and university groups in New Mexico and Pennsylvania who have shown interest in having their clients use Buoy, one of which is currently testing. @astridw and I were in the meeting. 

I think one way to resolve this would be to have a **disclaimer** that pops up when you first register for Buoy, which will (roughly) state that users will be responsible for how they use Buoy, that technology has its moments of failure such as outages, loss of connection etc thus it cannot be responsible for those circumstances, and that Buoy is not meant to keep you safe on its own: it is the responsibility of those in your community of friends and family to respond, not an organization **unless otherwise stated** and it is an option to use this alternative crisis response system.

Basically potential and current users and admins we've spoken to want it to be clear whether or not they are held responsible if someone gets hurt, for example, if they did not answer an alert in time and something disastrous happens as a result. 

Similar enhancements and info #4 - #33 - [Concerns About Misuse](https://github.com/betterangels/buoy/wiki/Frequently-Asked-Questions#concerns-about-misuse)

**Comment by fabacab** [comment]
This feels out of scope to me. Individual orgs can create their own disclaimers as part of their user registration process. This isn't something we need to consider as part of Buoy itself.
